src/Steps/Release/UpdateTranslations.php

```diff
@@ -295,16 +310,12 @@ protected function generateJavascript(OutputInterface $output, $modules)
      * @param OutputInterface $output
      * @param Module[] $modules
      */
-    public function pushSource(OutputInterface $output, $modules)
+    public function transifexPushSource(OutputInterface $output, $modules)
```

Comment:
There's no outside reference to the method. Might as well make it protected to make clear we don't expect outside classes to call it.
```suggestion
    protected function transifexPushSource(OutputInterface $output, $modules)
```

Reply:
Updated